options/internal/include/bits/size_t.h

```diff
@@ -1,7 +1,6 @@
 #ifndef MLIBC_SIZE_T_H
 #define MLIBC_SIZE_T_H
 
-#define __need_size_t
-#include <stddef.h>
+typedef __SIZE_TYPE__ size_t;
```

Comment:
Won't think break if somebody includes both this file (indirectly) and GCC's `stddef.h`?

Reply:
As long as they are the same type, which they should be else `stddef.h` would be broken, it should not break.